python/paddle/fluid/layers/nn.py

```diff
@@ -974,7 +974,12 @@ def cos_sim(X, Y):
     return out
 
 
-def dropout(x, dropout_prob, is_test=False, seed=None, name=None):
+def dropout(x,
+            dropout_prob,
+            is_test=False,
+            seed=None,
+            name=None,
+            div_prob_in_train=False):
```

Comment:
Betther to tell users the calculation formulas when set div_prob_in_train True  and False. And in the doc, tell users when set True, the behavior is the same as TensorFlow ?